utilities_common/netstat.py

```diff
@@ -56,6 +56,42 @@ def ns_util(newstr, oldstr, delta, port_rate=PORT_RATE):
         util = rate/(port_rate*1000*1000*1000/8.0)*100
         return "{:.2f}%".format(util)
 
+def format_brate(rate):
+    """
+    Show the byte rate.
+    """
+    if rate == STATUS_NA:
+        return STATUS_NA
+    else:
+        rate = float(rate)
+        if rate > 1024*1024*10:
+            rate = "{:.2f}".format(rate/1024/1024.0)+' MB'
+        elif rate > 1024*10:
+            rate = "{:.2f}".format(rate/1024.0)+' KB'
+        else:
+            rate = "{:.2f}".format(rate)+' B'
+        return rate+'/s'
+
+def format_prate(rate):
+    """
+    Show the packet rate.
+    """
+    if rate == STATUS_NA:
+        return STATUS_NA
+    else:
+        #TODO what does it mean /s maybe p/s packets per seconds
+        return "{:.2f}".format(float(rate))+'/s'
+
+def format_util(brate, port_rate=PORT_RATE):
+    """
+    Calculate the util.
+    """
+    if brate == STATUS_NA:
+        return STATUS_NA
+    else:
+        util = brate/(float(port_rate)*1024*1024*1024/8.0)*100
```

Comment:
Should we assume that speed is always in Gbps? I believe here we convert it from Mbps to Gpbs and pass it on [here](https://github.com/Azure/sonic-utilities/blob/6358537ae772ff2a17715b5959ac233243a8de8e/scripts/portstat#L218). But in future if we have a system which has deployment less than 1Gpbs this logic will fail. Instead I propose to use the default measurement unit used in the port table (Mbps) and modify the calculation accordingly here.

Reply:
already fixed